buildscripts/ci/linux/setup.sh

```
@@ -29,7 +29,7 @@ BUILD_TOOLS=$HOME/build_tools
 ENV_FILE=$BUILD_TOOLS/environment.sh
 PACKARCH="x86_64" # x86_64, aarch64, wasm
 COMPILER="gcc" # gcc, clang
-EMSDK_VERSION="4.0.7" # for Qt 6.10
+EMSDK_VERSION="4.0.7" # for Qt 6.11.2 (https://doc.qt.io/qt-6/wasm.html)
```

Shellcheck (0.11.0)

[warning] EMSDK_VERSION appears unused. Verify use (or export if used externally).

(SC2034)